src/sccz80/codegen.c

```diff
@@ -5610,7 +5627,84 @@ void gen_interrupt_leave(SYMBOL *func)
     nl();
 }
 
+void gen_ti_multipage_enter(void){
+        int from_bcall = getlabel();
+        int not_from_bcall = getlabel();
+
+        // ol("di");
+        ol("exx");                    
```

Comment:
I'm not sure how frequently this sequence is likely to be seen, but can it be pushed into a library routine (see the handling of `l_debug_push_frame`/`l_debug_pop_frame`) that way if it's used more than once the size is reduced and it becomes a bit more generic and the mechanism could be used for other targets.

Reply:
As this is something that would be used often, it would make sense to moved to a library routine. Although I doubt it would useful to other targets due to the use of the Floating Point Stack.

Reply:
That's fine - it can be located in a TI specific library with a generic name: `l_fixup_banked_prologue`etc